tools/MVP/templates/view.py.in

```diff
@@ -4,25 +4,23 @@
 #   NScD Oak Ridge National Laboratory, European Spallation Source,
 #   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
 # SPDX - License - Identifier: GPL - 3.0 +
-from qtpy.QtWidgets import QLabel, QPushButton, QVBoxLayout, QWidget
+from os import path
+from qtpy.QtWidgets import QWidget
+from qtpy import uic
 
 
 class View(QWidget):
-
     def __init__(self):
         super().__init__()
         self._presenter = None
 
-        self._layout = QVBoxLayout()
-        self._button = QPushButton("Increment", self)
-        self._label = QLabel("0")
-
-        self._layout.addWidget(self._button)
-        self._layout.addWidget(self._label)
-
-        self.setLayout(self._layout)
+        # Load gui from ui file.
+        # Mantid has a custom function to do the same: load_ui(fullpath, "ui_filename.ui")
+        # from mantidqt.utils.qt import load_ui
+        ui_path = path.dirname(path.realpath(__file__))
+        uic.loadUi(path.join(ui_path, "GUIWidget.ui"), self)
```

Comment:
Would it be worth adding a comment here to say something like "Note that this assumes the .ui file is in the same directory as the view file"?